electrum/wallet.py

```diff
@@ -1845,6 +1845,17 @@ def get_new_sweep_address_for_channel(self) -> str:
         assert is_address(selected_addr), f"not valid bitcoin address: {selected_addr}"
         return selected_addr
 
+    def should_keep_reserve_utxo(self, to_be_sent: int, is_channel_opening: bool) -> bool:
+        if self.lnworker and self.config.ENABLE_ANCHOR_CHANNELS:
```

Comment:
note that the config var config.ENABLE_ANCHOR_CHANNELS might get enabled then disabled, etc, by the user